procedures/auxtel/observation_procedures/Parabolic_Focus_Determination_Data_Acq.ipynb

```diff
@@ -9,23 +9,42 @@
     "\n",
```

Comment:
<p>Are these critical in the analysis? I like the idea of using your:</p><p> </p><pre class="ql-syntax" spellcheck="false">logger = logging.getLogger("SITCOM-627")
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/103/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>Removed these two less useful variables. </p>